airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java

```diff
@@ -176,7 +176,7 @@ protected boolean implementsAppend() throws WorkerException {
   protected boolean implementsAppendDedup() throws WorkerException {
     final ConnectorSpecification spec = runSpec();
     assertNotNull(spec);
-    if (spec.getSupportedDestinationSyncModes() != null) {
+    if (implementsBasicNormalization() && spec.getSupportedDestinationSyncModes() != null) {
```

Comment:
maybe move the `if implementsBasicNormalization` into the failing test (that is checking specifically for how normalization is doing it) instead of this `implementsAppendDedup` method

Reply:
It wasn't obvious to me a destination can implement normalisation without dedup. I'll remove it and leave a comment explaining this.